config/initializers/secure_headers.rb

```diff
@@ -66,11 +64,7 @@
     default_csp_config[:script_src] << "localhost:#{ENV['WEBPACK_PORT']}"
   end
 
-  if FeatureManagement.rails_csp_tooling_enabled?
-    config.csp = SecureHeaders::OPT_OUT
-  else
-    config.csp = default_csp_config
-  end
+  config.csp = SecureHeaders::OPT_OUT
```

Comment:
I think we probably want to start splitting out a second feature flag for `secure_headers` gem vs CSP? I re-used that feature flag for behavior outside of strictly the CSP header

I discovered in my testing for https://github.com/18F/identity-idp/pull/5952 that even with `config.csp = SecureHeaders::OPT_OUT`, the SecureHeaders gem still seems to stomp on default header configs:

> The way SecureHeaders is implemented, it stomps on the default_headers config very late in the game: https://github.com/github/secure_headers/blob/ce2ad139646f9775061159fe5c4707638fbe45c1/lib/secure_headers/railtie.rb#L21-L31


Another option would be to delay this PR until after we provide missing functionality in https://github.com/18F/identity-idp/pull/5963 which also uses this feature flag